examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,171 @@
+"""Fitting ODNP Datasets for Ksigma
+=================================== 
+The T1(p) and E(p) integrals are generated
+using the example, `generate_integrals.py`
+(as of 11/8/23 the `generate_integrals` is not
+updated), and stored in an H5 file.
+The inverse of the :math:`T_1(p)` data processing is taken
+to produce the :math:`R_1(p)` data which is fit using the
+polyfit of the inverse of krho with two degrees of
+freedom (*e.g.*, :math:`R_{100}(p) + \frac{C}{k_{\rho}^{-1}}`. 
```

Comment:
I think that you are saying:
```
so that :math:`R_1(p) = R_{1,0}(p) + \frac{C}{k_{\rho}^{-1}(p)}`
```

Reply:
in modified version, you need double backslash, or to put r in front of triple quotes.

Reply:
this was removed? why?